src/pilot/templates/nic-configs/dvr_6_port/nic_environment.yaml

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2018 Dell Inc. or its subsidiaries.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+resource_registry:
+  OS::TripleO::Controller::Net::SoftwareConfig: ./controller.yaml
+  OS::TripleO::Compute::Net::SoftwareConfig: ./compute.yaml
+  OS::TripleO::CephStorage::Net::SoftwareConfig: ./storage.yaml
+
+
+parameter_defaults:
+  # CHANGEME: Change the interface names in the following lines to include in
+  #           the controller nodes bonds
+  ControllerBond0Interface1: em1
+  ControllerBond0Interface2: p1p1
+  ControllerBond1Interface1: em2
+  ControllerBond1Interface2: p1p2
+  # The bonding mode to use for controller nodes
+  ControllerBondInterfaceOptions: 802.3ad miimon=100 xmit_hash_policy=layer3+4 lacp_rate=1
+
+  # CHANGEME: Change the interface names in the following lines to include in
+  #           the compute nodes bonds
+  ComputeBond0Interface1: em1
+  ComputeBond0Interface2: p1p1
+  ComputeBond1Interface1: em2
+  ComputeBond1Interface2: p1p2
+  ComputeBond2Interface1: p2p1
```

Comment:
Again, struggling to understand why we would add 2 1 port NICs instead of a single dual port NIC that's the same as the other dual port NIC in the box.